### PR TITLE
[DB]: added liquibase scripts

### PR DESCRIPTION
**Pull request by annton-shakur**
## Added liquibase scripts for creating the next tables:
* users
* roles
* users_roles
* amenities
* accommodations
* accommodations_amenities
* bookings
* payments
### for insertion data into tables:
* users
* roles
* amenities

### CODE REVIEW

File: src/main/resources/db/changelog/changes/07-create-amenities-table.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
```
Comment (vadymhrnk) [inline src/main/resources/db/changelog/changes/07-create-amenities-table.yaml:19]: I guess you forgot put unique here

**Review by vadymhrnk — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changes/10-create-accommodations-amenities-table.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      author: anton dudar
+      id: 09-create-accommodations-amenities-table
```
Comment (MinoDentori) [inline src/main/resources/db/changelog/changes/10-create-accommodations-amenities-table.yaml:4]: Explain pls, file name  10-create-accommodations-amenities-table.yaml, but id 09-create-accommodations-amenities-table?

File: src/main/resources/db/changelog/changes/11-create-bookings-table.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: 11-create-bookings-table
```
Comment (MinoDentori) [inline src/main/resources/db/changelog/changes/11-create-bookings-table.yaml:3]: also, you didn't use numbers in IDs for the first nine files.

**Review by MinoDentori — CHANGES_REQUESTED** [review]

**Review by MinoDentori — APPROVED** [review]
GJ

**Review by vadymhrnk — APPROVED** [review]
Nice work!

File: src/main/resources/db/changelog/changes/03-create-roles-table.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: role_name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
```
Comment (DPotapov93) [inline src/main/resources/db/changelog/changes/03-create-roles-table.yaml:19]: I think, we need "unique" here

**Review by DPotapov93 — APPROVED** [review]
Looks good, but put "role_name = unique" in 03-create-roles-table

**Pull request merged**